anthem/cli.py

```diff
@@ -53,9 +57,21 @@ def main():
         'configuration file and -d for the database name. Those must be at '
         'the end of the arguments'
     )
+    parser.add_argument(
+        'odoo-data-path',
+        nargs=argparse.REMAINDER,
```

Comment:
`REMAINDER` is to take all the "unknown" remaining args, which will be passed down to odoo, only `odoo-args` should use it. This argument should probably only have `action='store_true'` and no `nargs`